src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  background-color: #fff;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  margin: 70px;
+  width: 100%;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-search-big {
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat ;
+  background-position: 25px;
+  border: 0;
+}
+
+.form-search-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-search-big:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.form-search-small {
+  padding-left: 30px;
+  font-size: 14px;
+  height: 45px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat ;
+  background-position: 11px;
+  background-size: 11px;
+}
+
+.form-search-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
```

Comment:
Don't repeat the same styles. Move them on common class

Reply:
there is differets value of shadow:
box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1); - to class = "form-search-small"
box-shadow: 0 3px 8px rgba(45, 95, 151, 0.2); - to hover of this class
i need to change this?

Reply:
sry, found my mistake